pkg/containerd/events.go

```diff
@@ -97,6 +112,20 @@ func (e *EventMonitor) MonitorPorts(ctx context.Context) {
 					continue
 				}
 
+				// We only need to perform this check
+				// when the namespaced network is enabled
+				// in the case of default network when the
+				// privileged service is disabled, we will
+				// not be using this event API.
```

Comment:
This is a bit of a run-on sentence (a full 5 lines!)
```suggestion
				// We only need to perform this check
				// when the namespaced network is enabled.
				// When using the default network without
				// the privileged service, we will
				// not be using this event API.
```
It's missing _why_ we're not using that event API, though.
Also, it doesn't say anything about default network + privileged service…